scala/build/Dockerfile.latest

```
@@ -0,0 +1,87 @@
+
+# RUN npm install -g npm@latest
+RUN n stable
+
+# Install AWS CLI
+RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip" && \
+    unzip awscliv2.zip && \
+    ./aws/install && \
+    rm -rf awscliv2.zip ./aws
+
+# Install Docker and Docker Compose
+RUN curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add - && \
```

Hadolint

[info] Delete the apt-get lists after installing something

(DL3009)